ide/languages.hcl/src/org/netbeans/modules/languages/hcl/grammar/g4/HCLExpressionParser.g4

```diff
@@ -21,8 +21,9 @@ options { tokenVocab = HCLLexer; }
 
 expression
     : exprTerm
-    | operation
-    | <assoc=right> expression QUESTION expression COLON expression
+    | <assoc=right> unaryOperator expression
+    | expression binaryOperator expression
```

Comment:
As far as I recall the order of entries here is what decides operator precedence. You may want to change:

```
| expression binaryOperator expression
```

Into

```
| expression orOperator expression
| expression andOperator expression
| expression eqOp expression
| expression compOp expression
(etc)
```

Following the operator precedence table at https://github.com/hashicorp/hcl/issues/553 

Otherwise the expression "1 + 2 / 3" may have different results.

Reply:
Thank you!  I had to lift all operators to expression level, for the adjustment.
Created a few tests as well. My own Terraform files are still parsing well.

Reply:
@lkishalmi Great job!